### PR TITLE
Add user authentication

### PR DESCRIPTION
**Pull request by andreydro**

### CODE REVIEW

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (kavalenka) [inline app/assets/stylesheets/sessions.scss]: Please delete empty file

File: spec/helpers/sessions_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
```
Comment (kavalenka) [inline spec/helpers/sessions_helper_spec.rb]: One more empty file

File: spec/views/sessions/new.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (kavalenka) [inline spec/views/sessions/new.html.erb_spec.rb]: Empty file :)

File: app/helpers/sessions_helper.rb [hunk 4]
```diff
@@ -0,0 +1,45 @@
+module SessionsHelper
+  # Logs in the given user
```
Comment (kavalenka) [inline app/helpers/sessions_helper.rb:2]: Where did you copy this code?)
Are you able to rewrite it by yourself?))

**Review by kavalenka — CHANGES_REQUESTED** [review]

File: app/helpers/sessions_helper.rb (hunk 4 above)
Comment (andreydro) [reply app/helpers/sessions_helper.rb:2]: There are parts of code I used on my previous project. It works fine anywhere

**Review by kavalenka — APPROVED** [review]

**Pull request merged**